eval_metrics.py

```diff
@@ -51,12 +53,16 @@ def read_dataset(dataset_name: str) -> ParsedDataset:
 def main(model_name: str, dataset_name: str):
     model = None
     dataset = None
+    encoder = common_utils.get_encoder("MiniLM")
     if model_name == "dumb":
```

Comment:
I guess we can move the model selection to `common_utils.py` like encoders and generators. Then we would be able to use it anywhere, and `eval_metrics.py` would be completely independent on tested models.

Reply:
I was hesitant on that, but we can try to do it like that.